### PR TITLE
Login

### PR DESCRIPTION
**Pull request by FlorRodriguezA**
## Summary
-I finished the login card

## Screenshot
Captura de pantalla de 2021-02-27 11-10-42

## Trello Card
https://trello.com/c/N1ak4TXY

### CODE REVIEW

File: login/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <meta charset="utf-8">
+    <title>Login</title>
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+
+  <body>
+    <div class="container">
+      <img  class ="logo-wolox" src="LogoWolox.png" alt="Logo Wolox">
+      <form>
+        <label for="email">Email</label><br>
```
Comment (mcallegari10) [inline login/index.html]: We don't usually use `br` tags. If you need space, use `margin` 😄

File: login/scss/detail.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
+body {
```
Comment (mcallegari10) [inline login/scss/detail.scss]: Don't style the `body` directly. If you need a new container, create a wrapper `div`

File: login/index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <input class="data" type="email" id="email" name="email"><br>
+        <label for="pwd">Password</label><br>
+        <input class="data" type="password" id="pwd" name="pwd"><br>
+        <div class="login">
+        <a href="../listado-de-libros/index.html">
```
Comment (mcallegari10) [inline login/index.html]: Remember the identation

File: login/index.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    <div class="container">
+      <img  class ="logo-wolox" src="LogoWolox.png" alt="Logo Wolox">
+      <form>
+        <label for="email">Email</label><br>
+        <input class="data" type="email" id="email" name="email"><br>
+        <label for="pwd">Password</label><br>
+        <input class="data" type="password" id="pwd" name="pwd"><br>
+        <div class="login">
+        <a href="../listado-de-libros/index.html"> 
+          <input class="login" type="button"  value="Log In">
```
Comment (mcallegari10) [inline login/index.html]: For submit buttons, we use the `button` element with a `submit` type

File: login/index.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        </a>
+        </div>
+        
+        <div class="border">
+          <input class="singup" type="submit" value="Sing Up">
```
Comment (mcallegari10) [inline login/index.html]: I think this is actually a link, what do you think?

File: login/scss/detail.scss [hunk 6]
```diff
@@ -0,0 +1,66 @@
+body {  
+  width: 100vw;
+}
+
+.container {
+  margin: 0 auto;
+  width: 100%;
+  max-width: 300px;
+  background-color: $wild-sand;
```
Comment (mcallegari10) [inline login/scss/detail.scss:13]: Remember the wrapper container trick we saw the other day, try to use that 😄

File: login/scss/detail.scss [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-wolox {
+  height: 70px;
+  width: 210px;
+  position: relative;
```
Comment (mcallegari10) [inline login/scss/detail.scss]: I think this is not needed

File: login/scss/detail.scss [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+  margin-bottom: 20px;
+}
+
+label {
```
Comment (mcallegari10) [inline login/scss/detail.scss]: Don't style HTML tags directly, create a class for this

File: login/scss/detail.scss [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+  height: 22px;
+  color: $black;
+  font-size: 16px;
+  line-height: 24px;
+  position: relative;
```
Comment (mcallegari10) [inline login/scss/detail.scss]: No need for this

File: login/scss/detail.scss [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+  margin-bottom: 10px;
+}
+
+.data {
```
Comment (mcallegari10) [inline login/scss/detail.scss]: I think this is not a good name 🤔 . Maybe something like `input`?

File: login/scss/detail.scss [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+  height: 32px;
+  width: 252px;
+  border-radius: 5px;
+  background-color: $white;
+  position: relative;
```
Comment (mcallegari10) [inline login/scss/detail.scss]: Not needed

File: login/scss/detail.scss [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+  height: 32px;
+  width: 252px;
+  border-radius: 5px;
+  background-color: $earls-green;
+  position: relative;
+  margin-bottom: 15px;
+  
+}
+
+.border {
```
Comment (mcallegari10) [inline login/scss/detail.scss]: Why a new `div` for this? I think you can achieve the same with just the original element 😄

File: login/scss/detail.scss [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+.singup {
+  box-sizing: border-box;
+  height: 32px;
+  width: 252px;
+  border-radius: 5px;
+  background-color: $wild-sand;
+  border: 2px solid $earls-green;
+  position: relative;
+  margin-bottom: 20px;
+  margin-top: 15px;
```
Comment (mcallegari10) [inline login/scss/detail.scss]: Remember to use the shorthand

File: login/scss/equalizer.scss [hunk 14]
```diff
@@ -0,0 +1,92 @@
+$equalizer-black: #000;
```
Comment (mcallegari10) [inline login/scss/equalizer.scss:1]: Why a new equalizer?

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

**Comment by FlorRodriguezA** [comment]
Summary
-I finished the login card

Screenshot
Captura de pantalla de 2021-03-02 15-03-42

Trello Card
https://trello.com/c/N1ak4TXY

File: login/index.html [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+  <body>
+    <div class="login-container">
+      <div class="container">
+        <img  class ="logo-wolox" src="LogoWolox.png" alt="Logo Wolox">
+        <form class="form">
+          <label for="email">Email</label>
+          <input class="input" type="email" id="email" name="email">
+          <label for="pwd">Password</label>
+          <input class="input" type="password" id="pwd" name="pwd">
+          <button class="login" type="submit" formaction="../listado-de-libros/index.html" value="Log In">Log In</button>
```
Comment (mcallegari10) [inline login/index.html]: The `formaction` is not needed 😄

File: login/scss/detail.scss (hunk 8 above)
Comment (mcallegari10) [reply login/scss/detail.scss]: Ping here

File: login/scss/detail.scss (hunk 9 above)
Comment (mcallegari10) [reply login/scss/detail.scss]: Ping here

**Review by mcallegari10 — APPROVED** [review]
Check those comments and ready to merge 🚀 . Great job!

**Pull request merged**